src/server_manager/ui_components/outline-manual-server-entry.html

```diff
@@ -246,119 +360,193 @@
+          hidden$="[[!showConnection]]"
+          indeterminate
+          class="slow"
+        ></paper-progress>
       </div>
     </outline-step-view>
   </template>
   <script>
     Polymer({
-      is: 'outline-manual-server-entry',
+      is: "outline-manual-server-entry",
       properties: {
+        installScriptStepNumber: {
+          type: Number,
+          value: 2
+        },
+        pasteJsonStepNumber: {
+          type: Number,
+          value: 3
+        },
         placeholderText: {
           type: String,
-          value: '{"apiUrl":"https://xxx.xxx.xxx.xxx:xxxxx/xxxxxxxxxxxxxxxxxxxxxx","certSha256":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"}'
-
+          value:
+            '{"apiUrl":"https://xxx.xxx.xxx.xxx:xxxxx/xxxxxxxxxxxxxxxxxxxxxx","certSha256":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"}'
         },
         showConnection: Boolean,
         cloudProvider: {
           type: String,
-          value: 'generic'
+          value: "generic",
+          observer: "_cloudProviderChanged"
         },
         cloudProviderName: {
           type: String,
-          computed: '_computeCloudProviderName(cloudProvider)'
+          computed: "_computeCloudProviderName(cloudProvider)"
         },
         isCloudProviderAws: {
           type: Boolean,
-          computed: '_computeIsCloudProviderAws(cloudProvider)'
+          computed: "_computeIsCloudProviderAws(cloudProvider)"
         },
         isCloudProviderGcp: {
           type: Boolean,
-          computed: '_computeIsCloudProviderGcp(cloudProvider)'
+          computed: "_computeIsCloudProviderGcp(cloudProvider)"
         },
         isGenericCloudProvider: {
           type: Boolean,
-          computed: '_computeIsGenericCloudProvider(cloudProvider)'
+          computed: "_computeIsGenericCloudProvider(cloudProvider)"
         },
         enableDoneButton: {
           type: Boolean,
-          value: false,
+          value: false
         },
         localize: {
           type: Function,
-          readonly: true,
-        },
+          readonly: true
+        }
       },
       doneTapped: function() {
         this.showConnection = true;
-        this.fire('ManualServerEntered', {
+        this.fire("ManualServerEntered", {
           userInput: this.$.serverConfig.value
         });
       },
       cancelTapped: function() {
-        this.fire('ManualServerEntryCancelled');
+        this.fire("ManualServerEntryCancelled");
       },
       retryTapped: function() {
         this.showConnection = false;
         this.doneTapped();
       },
       clear: function() {
-        this.$.serverConfig.value = '';
+        this.$.serverConfig.value = "";
         this.showConnection = false;
         this.$.awsDropDown.hide();
-        this.$.gcpDropDown.hide();
+        // TODOBEFOREPUSH use $$ to select all drop-downs
+        for (const dropdown of Polymer.dom(this.root).querySelectorAll("iron-collapse")) {
```

Comment:
This is a bit fragile. It's hiding all iron-collapse things in the component tree. 
You should group things instead. Or maybe use some selector instead.